app/src/app.config.json.tpl

```diff
@@ -6,6 +6,7 @@
   "providers": "${APP_CONFIG_PROVIDER}",
   "authType": "${APP_CONFIG_AUTH_TYPE}",
   "loginRoute": "login",
+  "aosPlugin" : ${APP_CONFIG_PLUGIN_AOS},
```

Comment:
the position is completely wrong. There is a specific section that contains all the plugins

Reply:
i.e. 

"plugins": {
    "microsoftOnline": "${APP_CONFIG_PLUGIN_MICROSOFT_ONLINE}",
    "processAutomation": true,
    "aiService": ${APP_CONFIG_PLUGIN_AI_SERVICE}
  },

Reply:
I couldn't find that section in the `app.config.json.tpl` file but I agree, I will add that section for this **aosPlugin** variable.